interface/admin.py

```diff
@@ -133,22 +139,68 @@ def zip_submissions(self, request, submissions):
             review_zip = (tmp / 'review.zip').open('rb')
             return FileResponse(review_zip)
 
-    @log_action('Download last submissions')
-    def download_review_submissions(self, request, queryset):
+    @log_action('Run moss check')
+    def run_moss(self, request, queryset):
```

Comment:
There's a lot of code in this admin view. How about you move it to another module, like `moss.py`?